runtime/vm/JFRChunkWriter.cpp

```diff
@@ -77,10 +78,12 @@ U_8 *
 VM_JFRChunkWriter::writeJFRMetadata()
 {
 	/* reserve metadata size field */
-	U_8 *dataStart = _bufferWriter->getAndIncCursor(sizeof(U_32));
+	U_8 *dataStart = reserveEventSize();
 
 	if (_debug) {
-		j9tty_printf(PORTLIB, "Metadata frame start offset = 0x%lX\n", _bufferWriter->getFileOffsetFromStart(dataStart));
+		j9tty_printf(
+			PORTLIB, "Metadata frame start offset = 0x%" OMR_PRIX64 "\n",
+			_bufferWriter->getFileOffsetFromStart(dataStart));
```

Comment:
When folding, each argument goes on a line by itself (indented two tabs, not one).